app/packs/src/models/Attachment.js

```diff
@@ -36,6 +37,11 @@ export default class Attachment extends Element {
     this._preview = preview;
   }
 
+  static buildCopy(_attachments) {
+    const newAttachments = cloneDeep(_attachments);
```

Comment:
Cloning of an attachment is a little bit tricky because you do not only want to copy the JS element but maybe also the corresponding file. I think here we should discuss about the future approach/goal of the copy method.